backend/src/schema.rs

```diff
@@ -1,9 +1,28 @@
 // @generated automatically by Diesel CLI.
```

Comment:
Can we avoid having this file checked in? It gets generated when running migrations anyway?